moonraker/common.py

```diff
@@ -109,6 +109,7 @@ def __str__(self) -> str:
         return self._name_.lower()  # type: ignore
 
 class JobEvent(ExtendedEnum):
+    LAYER_CHANGED = 0
```

Comment:
Wouldn't it be better to append the new event to the bottom of the Enum with a value of `8`?

Reply:
I did spot this when I was coding it originally. Setting it to 8 would require additional changes a few lines further down in common.py which determines if the printer has finished or aborted. Adding the new state as 0 (zero) didn't disturb that existing code. 

```
    @property
    def finished(self) -> bool:
        return self.value >= 5

    @property
    def aborted(self) -> bool:
        return self.value >= 6
```

We could change it to this *untested* code, but I don't know if that might cause any problems elsewhere.

```
    LAYER_CHANGED = 8

    @property
    def finished(self) -> bool:
        return (self.value >= 5 and self.value != 8)

    @property
    def aborted(self) -> bool:
        return (self.value >= 6 and self.value != 8)

```